doc/release/2.11.0.rst

```diff
@@ -0,0 +1,586 @@
+Tarantool 2.11.0
+================
+
+Released on 2023-05-22.
+
+*   Release: :tarantool-release:`2.11.0`.
+*   Tag: ``2.11.0``.
+
+
+The 2.11 release of Tarantool includes many new features and fixes for Enterprise and Community versions. 2.11 is the long-term support (LTS) release with four years of maintenance. This means that you will continue to receive all the necessary security fixes and bug fixes throughout this period. You can learn more about the Tarantool release policy from the :doc:`corresponding document </release/policy>`.
+
+
+
+.. _enterprise:
+
+Enterprise
+----------
+
+
+.. _security_enhancements:
+
+Security enhancements
+~~~~~~~~~~~~~~~~~~~~~
+
+
+.. _encrypted_ssl_tls_keys:
+
+Encrypted SSL/TLS keys
+**********************
+
+Tarantool Enterprise now supports `encrypted <https://www.tarantool.io/en/enterprise_doc/security/#traffic-encryption>`_ SSL/TLS private key files protected with a password.
+Given that most certificate authorities generate encrypted keys, this feature simplifies the maintenance of Tarantool instances.
+
+A password can be provided using either the new ``ssl_password`` URI parameter or in a text file specified using ``ssl_password_file``, for example:
+
+.. code-block:: lua
+
+    box.cfg{ listen = {
+        uri = 'localhost:3301',
+        params = {
+            transport = 'ssl',
+            ssl_key_file = '/path_to_key_file',
+            ssl_cert_file = '/path_to_cert_file',
+            ssl_ciphers = 'HIGH:!aNULL',
+            ssl_password = 'topsecret'
+        }
+    }}
+
+
+.. _security_enforcement_options:
+
+Security enforcement options
+****************************
+
+With v2.11, Tarantool Enterprise includes new security enforcement options.
+These options provide the ability to improve security by flexible customization of a password policy.
+A password policy enables you to enforce the use of strong passwords, set up a maximum password age, and so on.
+For example, the ``password_min_length`` configuration option specifies the minimum number of characters for a password:
```

Comment:
This whole description seems a bit too wordy and detailed for a What's New page.
To me, this would be enough:
```
A password policy enables you to enforce the use of strong passwords, set up a maximum password age, and so on.
A couple examples:
* the minimum password length:
  #code#
*  the maximum password lifetime: 
  #code#   
Note that by default, new options are not specified.
You can learn more about all the available options from ...
```